SU-SOE/stanford_soe_helper
# Getting things ready for push to stage

# READY FOR REVIEW

# Summary
- Update all version numbers in .info files to 7.x-2.1-alpha1 (WM)
- Update CHANGELOG.txt with a human-readable version of what’s been changed since 7.x-2.0-beta3 (WM)


# Needed By (04/02/2018)

# Criticality
- Client is eagerly awaiting this.

# Steps to Test
- Make sure all the .info files and CHANGELOG.txt files are correct with the 7.x-2.1-alpha1 update

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-2775
https://stanfordits.atlassian.net/browse/SOE-2606

## More Information

## Folks to notify
@minorwm, @jbickar
